.github/workflows/ci.ml

```diff
@@ -0,0 +1,65 @@
+module List = ListLabels
+
+let skip_test =
+  match Sys.getenv "SKIP_TEST" with
+  | exception Not_found -> false
+  | s -> bool_of_string s
+
+let run cmd args =
+  let cmd = String.concat " " (cmd :: args) in
```

Comment:
This is extremely brittle (see `Filename.quote_command`) - it's worth at least a comment